README.md

```diff
@@ -147,7 +141,7 @@ region_kms_keys = {
 ```
 
 ```console
-AWS_PROFILE=cool-images-ec2amicreate-windows-commando-packer packer build --timestamp-ui -var-file release.pkrvars.hcl src/packer.pkr.hcl
+AWS_PROFILE=cool-images-ec2amicreate-windows-server-packer packer build --timestamp-ui -var-file release.pkrvars.hcl src/packer.pkr.hcl
```

Comment:
The example build command should be updated to reflect that you _must_ pass a value for `winrm_password`.

Reply:
@mcdonnnj resolved with [a7dae0d](https://github.com/cisagov/windows-server-packer/pull/10/commits/a7dae0df1dc23fa92edf9e0000d303e6582d7a8b)

Reply:
@mcdonnnj resolved with [46c6ab2](https://github.com/cisagov/windows-server-packer/pull/10/commits/46c6ab255bd389117c5663ab2b288b092cf80c4f)